samples/test-vulnerable-20260130.py

```
@@ -0,0 +1,57 @@
+다양한 취약점 패턴 테스트용 샘플
+"""
+
+import subprocess
+import os
+import pickle
+
+# ========== SAST 취약점 (Semgrep) ==========
+
+# 1. Command Injection (CRITICAL)
+def run_command(user_input):
+    os.system(f"echo {user_input}")  # shell injection
```

Ruff (0.14.14)

[error] Starting a process with a shell, possible injection detected

(S605)